README.md

```diff
@@ -12,125 +12,139 @@ Usage of this API is limited to registered users only. If you would like to use
 Run the following command to install the package using composer:
 
 ```
-$ composer require oberonamsterdam/tor-graphql-client
+$ composer require oberonamsterdam/tor-api-client
 ```
 
 # Usage
 
-To use this Client you need to define the API Key and endpoint as an .env variable or supply it when initiating the client class.
-```
-TOR_GRAPHQL_ENDPOINT="https://travelbase.nl/api/management/v2/graphql"
-TOR_GRAPHQL_APIKEY="MY_SECRET_APIKEY"
-```
+To use this Client you need to provide the API Key and endpoint when initiating the client class.
 
 ```php
-$client = new \TOR\GraphQL\TorClient("https://example.com", "MY_SECRET_APIKEY");
+$client = new \Oberon\TorClient\ApiClient("https://travelbase.nl", "APIKEY");
```

Comment:
Misschien wel beter om "example.com" (of zo iets) te laten staan? Dit suggereert een beetje dat dit ook daadwerkelijk de juiste URL is, terwijl je de URL van een specifiek eiland/instantie nodig zal hebben om er echt mee te werken.

Reply:
- [x] wijzigen url naar example.com